connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/utils/ProtobufUtils.scala

```diff
@@ -229,14 +229,18 @@ private[sql] object ProtobufUtils extends Logging {
   }
 
   def readDescriptorFileContent(filePath: String): Array[Byte] = {
-    try {
-      FileUtils.readFileToByteArray(new File(filePath))
-    } catch {
-      case ex: FileNotFoundException =>
-        throw QueryCompilationErrors.cannotFindDescriptorFileError(filePath, ex)
-      case ex: NoSuchFileException =>
-        throw QueryCompilationErrors.cannotFindDescriptorFileError(filePath, ex)
-      case NonFatal(ex) => throw QueryCompilationErrors.descriptorParseError(ex)
+    if (filePath == null || filePath.isEmpty) {
```

Comment:
When does this happen? Would it be a bug?

Reply:
Ah, I think we should just revert this change. There was a bug when accepting the empty value for the third argument at the first time, but it has been fixed by addressing constructors.